compiler/src/iree/compiler/Dialect/Flow/Transforms/AnnotateDispatches.cpp

```diff
@@ -162,6 +162,19 @@ static std::string getOpNameWithoutDialectName(Operation *op) {
 }
 
 static std::string summarizeLinalgOp(linalg::LinalgOp op) {
+  // Check if the body only contains a yield.
+  bool hasOnlyYield = true;
+  op.walk([&](Operation *op) {
+    if (isa<linalg::YieldOp>(op))
+      return WalkResult::advance();
+    hasOnlyYield = false;
+    return WalkResult::interrupt();
+  });
+
+  if (hasOnlyYield) {
+    return "slow_memcpy";
```

Comment:
Sounds like (from the [ongoing discussion on Discord](https://discord.com/channels/689900678990135345/908461811026767952/1179845558668644413)) we may want to differentiate between a pure copy and some other data operations like transposes, by looking at the indexing maps. We could do that in a follow-up, but would this label be accurate or confusing for a transpose?

Reply:
Please do not make dispatch name be `slow_memcpy` when they are transpose ops... I think they are really just transpose or broadcast ops, but not `slow_memcpy`.

It is a copy op only if (a) the linalg op has a single yield op in the body and (b) all the indexing maps are identity. If I understand correctly, this kind of op are folded away in tensor's world.

It's better if you can share the IR (or add it to the lit test).

Reply:
Fixed